spec/factories/recipe.rb

```diff
@@ -0,0 +1,61 @@
+FactoryBot.define do
+  factory :scrambled_eggs, class: Recipe do
```

Comment:
Can we use traits for defining different types of recipes? Similar to what we have in `ingridients.rb`.